### ISSUE TITLE
Unable to install on Linux Mint 18.3 (Sylvia)

### ISSUE THREAD

**Issue by mimmus** [asbru-cm/asbru-cm#584]
**Describe the bug**
I'm unable to install asbru-cm on Linux Mint 18.3 (Sylvia), even enabling snapshot repo.
Problem is always 'libgtk3-simplelist-perl'.

**To Reproduce**
1. Set standard and snapshot repo:
```
curl -s https://packagecloud.io/install/repositories/asbru-cm/asbru-cm/script.deb.sh | sudo bash
curl -s https://packagecloud.io/install/repositories/asbru-cm/snapshots/script.deb.sh | sudo bash

```
2. Run:
`sudo apt install asbru-cm`
3. See error:
```
$ sudo apt install asbru-cm 
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 asbru-cm : Depends: libgtk3-simplelist-perl but it is not installable
E: Unable to correct problems, you have held broken packages.
```
**Environment (please complete the following information):**
 - OS: Linux Mint 18.3 (Sylvia)
 - Ásbrú Version I don't know (latest?)

I read about similar problems in the past but I was unable to solve.

**Comment by popxunga** [comment]
Hi ...
The package libgtk3-simplelist-perl is available on all Mint Releases I used/tested so far.
Maybe your repository data is outdated and/or corrupted.
In order to fix it, try the following:
sudo apt clean
sudo apt update
sudo apt upgrade
sudo apt install asbru-cm

Let us know the result.

**Comment by mimmus** [comment]
Nothing to do :(
What else can I do? Sorry but I have experience with Red Hat (RPM) packages,

**Comment by popxunga** [comment]
I do use both, Debian and RPM based distros, but I use one RPM distro in my daily job.
I have the package in my Mint 19.3 virtual machine, but I'm not sure from where I got it. Need to check.
I think that you may try grab the package from 19.3 repository and install it manually.

**Comment by mimmus** [comment]
Problem is always "libgtk3-simplelist-perl": it is not available anywhere or, at least, I was not able to find it.

**Comment by gfrenoy** [comment]
Please check #405 

```libgtk3-simplelist-perl``` is not available in Ubuntu 16.04 (or any other distro based on that distribution) ; you'll need to add an additional repository ; see the mentioned issue for details.

**Comment by mimmus** [comment]
Solved manually installing libgtk3-simplelist-perl from PPA!

Thanks
